ui/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.0.0",
   "type": "module",
   "engines": {
-    "node": "21.1.0"
+    "node": "22.15.0"
```

Comment:
This is required for current vite since node 21.1.0 is EoL
Jump to current LTS version of 22.x, current release is 22.15.0